File: 807a/main.py
def I():
    return input()


def II():
    return int(input())


def MII():
    return map(int, input().split())


def LI():
    return list(input().split())


def LII():
    return list(map(int, input().split()))

# alpha = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']

a = II()
rate = False
Maybe = False
prev = float('inf')
for _ in range(a):
    b, c = MII()
    if not rate and b != c:
        rate = True
    if not Maybe and b > prev:
        Maybe = True
    prev = b

if rate:
    print("rated")
elif not Maybe:
    print("maybe")
else:
    print("unrated")

File: 807a/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	var a int
	fmt.Fscan(reader, &a)
	r := false
	m := false
	prev := 1 << 20
	var x, y int
	for i := 0; i < a; i++ {
		fmt.Fscan(reader, &x, &y)
		if !r && x != y {
			r = true
		}
		if !m && x > prev {
			m = true
		}

		prev = x
	}

	if r {
		fmt.Println("rated")
	} else if m {
		fmt.Println("unrated")
	} else {
		fmt.Println("maybe")
	}
}
